### DISCUSSION TITLE
repeating same routine, is a faster way possible ?

### DISCUSSION THREAD

**Discussion by benjaminfrombe** [mifi/lossless-cut#2021] [Support]
Hi,

I need to perform the same routine on a daily basis, which exists out: 

- (manually) removing the start en end from a video
- chopping out the resulting video in max 2GB segments

for now I always do this in 2 steps: 
- first exporting the segment with the start en end cut off
- then re-opening the newly created file and dividing it in equal parts smaller then 2GB (I look at the toal filesize and I manually choose the amount of segments, so if a file is 8,2GB I create 5 equal segments for example)
- then I export again

It seems ineffective to export twice, is there any way to do this differently ? Or should I create a feature request to be able to split a segment in equal segments (instead of the whole file) ?

**Comment by mifi** [comment]
I think for such a specific automation, I would just write a bash script that automates this using ffmpeg. You can look inside ffmpeg "last ffmpeg commands" list and use that as inspiration. chatgpt might get you a long way on implementing such a script.
